toy_robot.rb

```diff
@@ -7,34 +7,47 @@ def initialize
     @x = 0
     @y = 0
     @facing = :north
+    @movements = []
   end
 
   def place(x,y,facing)
     raise InvalidPositionError.new("Invalid Position, #{x}, #{y}, #{facing}") if !(x.between?(0,4)) || !(y.between?(0,4))
     @x = x
     @y = y
     @facing = facing
+    @movements.replace(["PLACE #{@x}, #{@y}, #{@facing.to_s.upcase}"])
   end
 
   def move
     if (@facing == :north) && (@y < 4)
       @y =+ 1
+      @movements << "MOVE"
     elsif (@facing == :south) && (@y > 0)
       @y =- 1
+      @movements << "MOVE"
     elsif (@facing == :east) && (@x < 4)
       @x =+ 1
+      @movements << "MOVE"
     elsif (@facing == :west) && (@x > 0)
       @x =- 1
+      @movements << "MOVE"
     end
   end
 
 end
 
 
-# robot = ToyRobot.new
-# puts robot.x
-# puts robot.y
-# puts robot.facing
+robot = ToyRobot.new
```

Comment:
put this code somewhere else. You should get used to using your Tests as your test harness. I understand this code is helpful but put it in a sample_code.rb or something.